### PR TITLE
Update interceptors.md

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes https://github.com/grpc/grpc-dotnet/issues/1630

### CODE REVIEW

File: aspnetcore/grpc/interceptors.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ The preceding code:
   * Calls the `continuation` parameter to invoke the next item in the interceptor chain.
   * Creates a new `AsyncUnaryCall<TResponse>` instance based on the result of the continuation.
   * Wraps the `ResponseAsync` task using the `HandleResponse` method.
-  * Awaits the response with `HandleResponse`.
+  * Awaits the response with `HandleResponse`. Awaiting the response allows logic to be added after the client received the response. By awaiting the response in a try/catch block, errors from the response can be logged.
```
Comment (JamesNK) [inline aspnetcore/grpc/interceptors.md:126]:
```suggestion
  * Awaits the response with `HandleResponse`. Awaiting the response allows logic to be added after the client received the response. By awaiting the response in a try/catch block, errors from the call can be logged.
```

File: aspnetcore/grpc/interceptors.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ The preceding code:
   * Calls the `continuation` parameter to invoke the next item in the interceptor chain.
   * Creates a new `AsyncUnaryCall<TResponse>` instance based on the result of the continuation.
   * Wraps the `ResponseAsync` task using the `HandleResponse` method.
-  * Awaits the response with `HandleResponse`.
+  * Awaits the response with `HandleResponse`. Awaiting the response allows logic to be added after the client received the response. By awaiting the response in a try/catch block, errors from the call can be logged.
```
Comment (JamesNK) [inline aspnetcore/grpc/interceptors.md:126]:
```suggestion
  * Awaits the response with `HandleResponse`. Awaiting the response allows logic to be added after the client received the response. By awaiting the response in a try/catch block, errors from calls can be logged.
```

File: aspnetcore/grpc/interceptors.md [hunk 3]
```diff
@@ -123,7 +123,7 @@ The preceding code:
   * Calls the `continuation` parameter to invoke the next item in the interceptor chain.
   * Creates a new `AsyncUnaryCall<TResponse>` instance based on the result of the continuation.
   * Wraps the `ResponseAsync` task using the `HandleResponse` method.
-  * Awaits the response with `HandleResponse`.
+  * Awaits the response with `HandleResponse`. Awaiting the response allows logic to be added after the client received the response. By awaiting the response in a try/catch block, errors from calls can be logged.
```
Comment (Rick-Anderson) [inline aspnetcore/grpc/interceptors.md:126]:
```suggestion
  * Awaits the response with `HandleResponse`. Awaiting the response allows logic to be added after the client received the response. By awaiting the response in a try-catch block, errors from calls can be logged.
```

File: aspnetcore/grpc/interceptors.md [hunk 4]
```diff
@@ -211,11 +211,11 @@ Overriding `UnaryServerHandler`:
 * Intercepts an incoming unary call.
 * Logs details about the call.
 * Calls the `continuation` parameter passed into the method. This invokes the next interceptor in the chain or the service handler if this is the last interceptor.
-* Logs any exceptions.
+* Logs any exceptions. Awaiting the continuation allows logic to be added after the service method has executed. By awaiting the continuation in a try/catch block, errors from methods can be logged.
```
Comment (Rick-Anderson) [inline aspnetcore/grpc/interceptors.md:214]:
```suggestion
* Logs any exceptions. Awaiting the continuation allows logic to be added after the service method has executed. By awaiting the continuation in a try-catch block, errors from methods can be logged.
```

**Review by Rick-Anderson — APPROVED** [review]

**Review by wadepickett — APPROVED** [review]
Looks good! I don't see anything to change beyond what Rick already suggested.

**Pull request merged**